core/thread/test/CMakeLists.txt

```diff
@@ -4,8 +4,16 @@
 # For the licensing terms see $ROOTSYS/LICENSE.
 # For the list of contributors see $ROOTSYS/README/CREDITS.
 
-ROOT_ADD_UNITTEST_DIR(Core Thread Hist)
+if (tbb OR builtin_tbb)
+   ROOT_ADD_UNITTEST_DIR(Core Thread Hist ${TBB_LIBRARIES})
 
-ROOT_ADD_GTEST(testTThreadedObject testTThreadedObject.cxx LIBRARIES Hist)
+   ROOT_ADD_GTEST(testTThreadedObject testTThreadedObject.cxx LIBRARIES Hist ${TBB_LIBRARIES})
```

Comment:
Why is this needed?  Isn't tbb already linked through libThread (and/or libImt)?

Reply:
Yes I think you're right, will remove it.